Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM jakechampion/heroku-nodejs
```

Comment:
It uses an out-of-date nodejs. My version is identical bar the version of nodejs.